tests/model_serving/conftest.py

```diff
@@ -0,0 +1,6 @@
+import pytest
```

Comment:
It is not related to this PR, consider splitting it.

Reply:
already done in another pr